tests/test_hydrology.py

```diff
@@ -23,6 +28,38 @@ def test_simple(self, q_series):
         np.testing.assert_array_equal(out, 2)
 
 
+class TestStandardizedStreamflow:
+    nc_ds = Path("Raven", "q_sim.nc")
```

Comment:
This is really a nitpick, but you could probably just throw this into the test function